src/components/CurrencyInputPanel/index.tsx

```diff
@@ -259,6 +259,7 @@ export default function CurrencyInputPanel({
                 setModalOpen(true)
               }
             }}
+            labelOnly={!onCurrencySelect}
```

Comment:
following the above recommendation this would be 
```suggestion
            pointerEvents="none"
```

Reply:
Renamed